senders/beanstlakd_sender.go

```diff
@@ -31,7 +37,17 @@ func (c *BeanstalkdSender) SendEvent(ev interfaces.IncomingEventInterface, dv in
 		return &captin_errors.UnretryableError{Msg: "Event control is empty", Event: e}
 	}
 
-	conn, err := beanstalk.Dial("tcp", e.Control["beanstalkd_host"].(string))
+	beanstalkdHost := e.Control["beanstalkd_host"]
+	if beanstalkdHost == nil {
+		return &captin_errors.UnretryableError{Msg: "beanstalkd_host is empty", Event: e}
```

Comment:
Suggest to create a panel to monitor , please check with @sl-abdullah how you could do it

Reply:
you means create a ganfana dashboard to monitor?